i6_models/assemblies/conformer/conformer.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+import torch
+from torch import nn
+
+# TODO probably used for SpecAugment
+from torchaudio.transforms import TimeMasking, TimeStretch, FrequencyMasking
+from dataclasses import dataclass
+
+from i6_models.config import ModelConfiguration
+
+from i6_models.parts.conformer.convolution import ConformerConvolutionV1
+from i6_models.parts.conformer.feedforward import ConformerPositionwiseFeedForwardV1
+from i6_models.parts.conformer.mhsa import ConformerMHSAV1
+
+
+@dataclass
+class ConformerFeedForwardV1Config(ModelConfiguration):
+    dim: int
+
+
+@dataclass
+class ConformerMHSAV1Config(ModelConfiguration):
+    pass
+
+
+@dataclass
+class ConformerConvV1Config(ModelConfiguration):
+    pass
+
+
+@dataclass
+class ConformerBlockV1Config(ModelConfiguration):
+    # hyperparameters
+    norm_dim: int
+
+    # nested configurations
+    ff_cfg: ConformerFeedForwardV1Config  # Configuration for ConformerPositionwiseFeedForwardV1
+    mhsa_cfg: ConformerMHSAV1Config  # Configuration for ConformerMHSAV1
+    conv_cfg: ConformerConvV1Config  # Configuration for ConformerConvolutionV1
```

Comment:
Why do the names of the configs differ from the names of the classes they are used for (at least according to the comments)?

Reply:
I can change that for now, but this will change anyways and depend on the name given in the other PRs.